.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,42 @@
+name: Push to Pypi
+
+on:
+  release:
+    types: [released, prereleased]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install --no-cache-dir -e .[complete]
+        pip list
+    - name: Build the func_adl wheel
+      env:
+        func_adl_version: ${{ github.ref }}
+      run: |
+        python setup_func_adl.py sdist bdist_wheel
+    - name: Publish a func_adl to PyPI
+      uses: pypa/gh-action-pypi-publish@v1.3.1
+      with:
+        user: __token__
+        password: ${{ secrets.pypi_password_func_adl }}
+    - name: Build the func_adl.ast wheel
+      env:
+        func_adl_version: ${{ github.ref }}
+      run: |
+        rm -rf dist/
+        python setup_func_adl_ast.py sdist bdist_wheel
```

Comment:
Ahah, this is why. Personally, I think I've come up with a solution that allows you to stay entirely in the standard packaging ecosystem; if you are interested in changing, I can show you how it works. Then you'll be able to use standard packaging tools and avoid [potentially deprecated distutils](https://www.python.org/dev/peps/pep-0632/).

Reply:
I would love to know - tho I won't do the change in this PR!